service/modules/lin-reg-module/conda.yaml

```diff
@@ -0,0 +1,9 @@
+name: tutorial
+channels:
+  - defaults
+dependencies:
```

Comment:
numpy?

Reply:
Don't think it's necessary based on sql-module, as per our conversation